torch/csrc/autograd/variable.cpp

```diff
@@ -145,56 +77,50 @@ void Variable::backward(
 
   std::vector<Variable> inputs;
   if (!gradient.has_value()) {
-    gradient = make_variable(at::ones_like(data()), /*requires_grad=*/false);
+    gradient = at::ones_like(*this);
+    as_variable_ref(*gradient).set_requires_grad(false);
```

Comment:
is this necessary?  I thought `ones_like` defaults to not requiring grad.

Reply:
Fixed.